src/Meilisearch/Extensions/HttpExtensions.cs

```diff
@@ -16,13 +17,14 @@ public static class HttpExtensions
         /// <param name="client">HttpClient.</param>
         /// <param name="uri">Endpoint.</param>
         /// <param name="body">Body sent.</param>
+        /// <param name="cancellationToken">The cancellation token for this call.</param>
         /// <typeparam name="T">Type of the body to send.</typeparam>
         /// <returns>Returns the HTTP response from the MeiliSearch server.</returns>
-        public static async Task<HttpResponseMessage> PostJsonCustomAsync<T>(this HttpClient client, string uri, T body)
+        public static async Task<HttpResponseMessage> PostJsonCustomAsync<T>(this HttpClient client, string uri, T body, CancellationToken cancellationToken = default)
```

Comment:
Can you rename the `cancellationToken` variable into `ct` to make it less verbose? :)

Reply:
I would not do it for two reasons :

`cancellationToken` is the name used by almost every projects (Dapper, MediatR, XUnit,...)

Since `cancellationToken` is an optional parameter, if you have to specify it, I think it's better to have a clear name.

With `ct`, it may be a cancellationToken or something else
```csharp
http.PostJsonCustomAsync<T>(uri: "https://domain.tld", 
                            body: obj,
                            ct: ct);
```

With `cancellationToken`, it's clear
```csharp
http.PostJsonCustomAsync<T>(uri: "https://domain.tld", 
                            body: obj,
                            cancellationToken: ct);
```

I'm not sure of the rule, but it says something like "you write the code once, you read it ten times".

Just let me know what you prefer and I'll fix it.

Reply:
You're right
I mainly would to avoid `cancellationToken: cancellationToken` being written too many times since it's "heavy", but it might be less clear